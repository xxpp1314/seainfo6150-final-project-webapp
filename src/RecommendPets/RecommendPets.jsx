import React from 'react';
import styles from "./RecommendPets.module.css";
import Bane from "../images/Bane.jpg";
import Milo from "../images/Milo.jpg";
import { Link } from "react-router-dom";

const RecommendPets = () => {
    return (
        <div>
            <h1 className={styles.title}>
                Find Your Suitable Pet Here
            </h1>
            <div className={styles.container}>
                <div className={styles.grid}>
                    <div className={styles.div}>
                        <img className={styles.image} src={Bane} alt=""/>
                        <Link className={styles.text} to="/dogs/Bane">Smart Dog: Bane</Link>
                    </div>
                    <div className={styles.div}>
                        <img className={styles.image} src={Milo} alt=""/>
                        <Link className={styles.text} to="/cats/Milo">Cute Cat: Milo</Link>
                    </div>
                </div>
            </div>
        </div>
    )
}

export default RecommendPets