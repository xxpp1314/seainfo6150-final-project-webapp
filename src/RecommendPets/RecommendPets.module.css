.container {
    padding: 2%;
    box-shadow: 0px 0px 5px 2px rgba(204, 0, 0, 0.5);    
    border: 0px solid grey;
    border-radius: 1%;
    display: grid;
    grid-template-rows: 1fr;
    justify-items: center;
  }
  
  .image{
    padding: 2%;
    border-radius: 2%;
    width: 86%;
    height:320px;
    margin: 1.5%;
    align-items: center;
    justify-content: center;
    box-shadow:0px 0px 10px 5px grey;
    border: 5px solid maroon;
  }
  
  .div{
    display: flex;
    flex-direction: column;
    text-align: center;
    align-items: center;
  }
  
  .grid{
    padding-top: 4%;
    display: grid;
    grid-template-columns: repeat(2,1fr);
  }
  
  @media screen and (min-width: 200px) {
    .grid {
      grid-template-columns: repeat(1, 1fr);
      padding-bottom: 5%;
    }
  }
  
  @media screen and (min-width: 800px) {
    .grid {
      grid-template-columns: repeat(2, 1fr);
      padding-bottom: 1%;
    }
  }
  
  
  .text {
    padding-top: 2%;
    color:  rgb(221, 193, 193);
    text-align: center;
    align-items: center;
    justify-content: center;
    text-decoration-line: none;
    font-family: 'Space Grotesk', sans-serif;
    font-weight: lighter;
  }
  
  .text:hover{
    color:rgb(160, 27, 27);
  }
  
  .title {
    padding: 0.2%;
    text-align: center;
    background-color:  rgb(221, 193, 193);
    color:rgb(83, 15, 15);
    text-transform: capitalize;
    font-family: 'Lora', serif;
  }
  