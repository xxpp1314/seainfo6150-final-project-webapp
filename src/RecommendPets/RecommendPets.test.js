import React from "react";
import { render } from "@testing-library/react";
import RecommendPets from "./RecommendPets.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders the FeaturedGifts Page correctly", () => {
  const { container } = render(
    <BrowserRouter>  
        <RecommendPets/>
    </BrowserRouter> 
  );
  expect(container).toMatchSnapshot();
});
