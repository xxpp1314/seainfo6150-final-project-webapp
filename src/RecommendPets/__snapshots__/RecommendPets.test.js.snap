// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the FeaturedGifts Page correctly 1`] = `
<div>
  <div>
    <h1
      class="title"
    >
      Find Your Suitable Pet Here
    </h1>
    <div
      class="container"
    >
      <div
        class="grid"
      >
        <div
          class="div"
        >
          <img
            alt=""
            class="image"
            src="Bane.jpg"
          />
          <a
            class="text"
            href="/dogs/Bane"
          >
            Smart Dog: Bane
          </a>
        </div>
        <div
          class="div"
        >
          <img
            alt=""
            class="image"
            src="Milo.jpg"
          />
          <a
            class="text"
            href="/cats/Milo"
          >
            Cute Cat: Milo
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
